package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@mdx-js/mdx": "^1.6.22",
     "@mdx-js/react": "^1.6.22",
-    "@pantheon-systems/pds-toolkit-react": "^1.0.0-dev.106",
```

Comment:
I reverted the commit that updated the design system because I was getting an error.